app/config/AppConfig.scala

```diff
@@ -65,7 +65,7 @@ class PbikAppConfig @Inject()(configuration: Configuration) extends AppConfig {
   lazy val urBannerToggle: Boolean = configuration.get[Boolean]("urBanner.toggle")
   override lazy val urBannerLink: String = configuration.get[String]("urBanner.link")
   override lazy val feedbackUrl: String = configuration.get[String]("feedback.url")
-  override lazy val signOut = s"$companyAuthHost/gg/sign-out/?continue=$feedbackUrl"
+  override lazy val signOut = s"$basGatewayHost/bas-gateway/sign-out-without-state/?continue=$feedbackUrl"
```

Comment:
Same here - need to implement 'cotinue_url' query parameter rather than 'continue'

Reply:
That only applies for sign in (which I do need to update) but you've highlighted the sign out which is fine! This is the [sign-out url](https://github.com/hmrc/bas-gateway-frontend/blob/master/conf/app.routes#L16).